### DISCUSSION TITLE
Why don't we add the Docker Compose installation instructions to the documentation?

### DISCUSSION THREAD

**Discussion by iFurySt** [milvus-io/milvus#33189] [Q&A and General discussion] [answered]
https://milvus.io/docs/install_standalone-docker.md

For the installation section in docs, I'm confused about where the `docker-compose.yaml` is. I finally found it in the release, like this https://github.com/milvus-io/milvus/releases/download/v2.3.15/milvus-standalone-docker-compose.yml

Is it possible to add this instruction or reference to the docs?

**Comment by xiaofan-luan** [comment, marked as answer]
https://milvus.io/docs/install_standalone-docker-compose.md

<img width="906" alt="image" src="https://github.com/milvus-io/milvus/assets/83447078/d6088573-ad02-4887-8b4b-26afcb8cf6d9">

switch at the top

**Reply by iFurySt** [reply]
ah, sorry. I didn't see it.
